<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Lakkappa" failures="3" tests="10" name="Test" time="28.008" errors="0" timestamp="2023-12-08T17:21:10 IST">
  <testcase classname="TestCases.Test2" name="test1" time="5.06"/>
  <testcase classname="TestCases.Test1" name="demo" time="5.161">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test2" name="test12" time="3.857">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.Test2.test12(Test2.java:30)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- test12 -->
  <testcase classname="TestCases.Test1" name="demo" time="3.82">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test1" name="demo" time="3.517">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test1" name="demo" time="3.615">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test1" name="demo" time="4.445">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test1" name="demo" time="4.888">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.Test1.demo(Test1.java:101)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="TestCases.Test1" name="log" time="0.0">
    <failure type="org.testng.TestNGException" message="
Cannot inject @Test annotated Method [log] with [class java.util.HashMap].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [log] with [class java.util.HashMap].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- log -->
  <testcase classname="TestCases.Test1" name="loginPage" time="2.457"/>
</testsuite> <!-- Test -->
